pkg/common/config/types.go

```diff
@@ -43,6 +43,9 @@ type Config struct {
 		// CnsRegisterVolumesCleanupIntervalInMin specifies the interval after which
 		// successful CnsRegisterVolumes will be cleaned up.
 		CnsRegisterVolumesCleanupIntervalInMin int `gcfg:"cnsregistervolumes-cleanup-intervalinmin"`
+		// VolumeMigrationCRCleanupIntervalInHours specifies the interval after which
+		// stale CnsVSphereVolumeMigration CRs will be cleaned up.
+		VolumeMigrationCRCleanupIntervalInHours int `gcfg:"volumemigration-cr-cleanup-intervalinhours"`
```

Comment:
Let's set this in Minutes, and we will default to 60 minutes.

Reply:
Done